libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -1,19 +1,35 @@
 <ng-container *ngIf="(readingList$ | async).length > 0; else empty">
   <div class="reading-list-item" *ngFor="let b of readingList$ | async">
+    <div>
+      <button *ngIf="!b?.finished" data-testing="mark-to-finish-reading-button" mat-icon-button color="magenta"
+        [attr.aria-label]="'Mark ' + b.title + ' as finished in reading list'"
+        (click)="markAsFinishedReadingFromList(b)">
+        <mat-icon>library_books</mat-icon>
+      </button>
+      <button *ngIf="b?.finished" data-testing="marked-as-finish-reading-button" mat-icon-button color="primary"
```

Comment:
The UX solution you came up with here looks good, but because the icons aren't universally recognizable for the actions, I would change the `aria-label` attributes for these buttons ("finish" and "remove) to `title` attributes, or use tooltips, so that users without screenreaders can see a hint about what they do